plugin/trino-accumulo/pom.xml

```diff
@@ -18,16 +18,6 @@
         <dep.curator.version>2.12.0</dep.curator.version>
     </properties>
 
-    <dependencyManagement>
```

Comment:
It looks like there was a reason to do this. Do you know why it was handled this way?